src/components/Example1.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import '../styles/example1.css';
+import { addData, getData } from '../utils/indexedDB';
+
+interface Certificate {
+  id?: number;
+  supplier: string;
+  certificateType: string;
+  validFrom: string;
+  validTo: string;
+}
+
+const certificateNames: Certificate[] = [
```

Comment:
Please move sample data of certificates also to data.ts instead of keeping it in component. Keeping stuff that you don't need in component, makes it hard to read and too complex.